cli/src/main/java/dev/starfix/Starfix.java

```diff
@@ -82,8 +82,19 @@ public int cloneCmd(@Parameters(index = "0") String url) {
         }
 
         try {
-            URI uri = new URI(url);
+            String filePath = "";
+            String branch = "";
+            if(isBlob(url))
+            {   // Example URL : https://github.com/starfixdev/starfix/blob/master/cli/pom.xml
```

Comment:
our url protocol starts with ide:// why would this kind of url be passed to starfix ?

Reply:
> our url protocol starts with ide:// why would this kind of url be passed to starfix ?

Have a look at this code at the starting of file  that parses the URL if it contains ide://:
https://github.com/starfixdev/starfix/blob/aa0a342e15aecbb719b6a6f3e230d1d480287922/cli/src/main/java/dev/starfix/Starfix.java#L46-L49

Reply:
but we can't keep doing that since the ide:// command would have something like ide://clone?repo=repourl etc.